test/core/fixtures/transformation/validation.undeclared-variable-check/undeclared-shorthand-property/exec.js

```diff
@@ -0,0 +1 @@
+({foo})
```

Comment:
The key and value are the exact same object in the case of a shorthand property. Is that a bug?